i18n/ko.json

```
@@ -1,1962 +1,1962 @@
+    "backup_keep_last_amount": "보관할 이전 덤프 수",
+    "backup_settings": "데이터베이스 덤프 설정",
+    "backup_settings_description": "데이터베이스 덤프 설정을 관리합니다. 참고: 이 작업은 상태를 모니터링하지 않으며 실패 시 알림이 없습니다.",
+    "check_all": "모두 선택",
+    "cleanup": "정리",
+    "cleared_jobs": "{job} 작업 정리됨",
+    "config_set_by_file": "현재 구성은 설정 파일에 의해 지정되어 있습니다.",
+    "confirm_delete_library": "{library} 라이브러리를 삭제하시겠습니까?",
+    "confirm_delete_library_assets": "이 라이브러리를 삭제하시겠습니까? Immich에서 {count, plural, one {항목 #개} other {항목 #개}}가 삭제되며 되돌릴 수 없습니다. 파일은 디스크에 남아 있습니다.",
+    "confirm_email_below": "계속하려면 아래에 \"{email}\"을 입력하세요.",
+    "confirm_reprocess_all_faces": "모든 얼굴을 다시 처리하시겠습니까? 이 작업은 지정된 인물 정보도 초기화합니다.",
+    "confirm_user_password_reset": "{user}님의 비밀번호를 재설정하시겠습니까?",
+    "confirm_user_pin_code_reset": "{user}님의 PIN 코드를 초기화하시겠습니까?",
+    "create_job": "작업 생성",
+    "cron_expression": "Cron 표현식",
+    "cron_expression_description": "Cron 형식을 사용해 스캔 주기를 설정합니다. 예시는 <link>Crontab Guru</link>를 참고하세요.",
+    "cron_expression_presets": "Cron 사전 설정",
+    "disable_login": "로그인 비활성화",
+    "duplicate_detection_job_description": "기계 학습으로 유사 이미지를 감지합니다 (스마트 검색 기능 필요).",
+    "exclusion_pattern_description": "라이브러리 스캔 시 제외할 파일 및 폴더를 정의합니다. 예: RAW 파일이 포함된 폴더 제외 등",
+    "external_library_created_at": "외부 라이브러리 (생성일: {date})",
+    "external_library_management": "외부 라이브러리 관리",
+    "face_detection": "얼굴 감지",
+    "face_detection_description": "기계 학습을 통해 항목에서 얼굴을 감지합니다. 영상은 썸네일만 사용됩니다. \"새로고침\"은 모든 항목을 재처리하고, \"초기화\"는 얼굴 데이터를 삭제합니다. \"누락됨\"은 아직 처리되지 않은 항목을 대기열에 추가합니다. 감지된 얼굴은 얼굴 인식 단계로 넘어갑니다.",
+    "facial_recognition_job_description": "감지된 얼굴을 인물별로 그룹화합니다. 이 작업은 얼굴 감지 완료 후 실행됩니다. \"초기화\"는 모든 얼굴을 재그룹화하고, \"누락됨\"은 미할당 얼굴을 대기열에 추가합니다.",
+    "failed_job_command": "{job} 작업에서 명령 {command}가 실패했습니다.",
+    "force_delete_user_warning": "경고: 이 작업은 사용자를 포함한 모든 항목을 즉시 삭제합니다. 복구할 수 없습니다.",
+    "forcing_refresh_library_files": "라이브러리 파일을 강제로 새로고침 중입니다.",
+    "image_format": "형식",
+    "image_format_description": "WebP는 JPEG보다 파일 크기가 작지만 인코딩 속도는 느립니다.",
+    "image_fullsize_description": "확대 보기 시 사용되는 메타데이터가 제거된 전체 크기 이미지",
+    "image_fullsize_enabled": "전체 크기 이미지 생성 활성화",
+    "image_fullsize_enabled_description": "웹에 적합하지 않은 포맷의 전체 크기 이미지를 생성합니다. \"내장 미리보기 우선 사용\"이 설정되어 있으면 변환 없이 내장 미리보기를 그대로 사용합니다. JPEG 등 웹 친화적 포맷에는 영향을 주지 않습니다.",
+    "image_fullsize_quality_description": "전체 크기 이미지 품질 (1~100). 높을수록 품질은 좋지만 파일 크기가 커집니다.",
+    "image_fullsize_title": "전체 크기 이미지 설정",
+    "image_prefer_embedded_preview": "내장 미리보기 우선 사용",
+    "image_prefer_embedded_preview_setting_description": "RAW 사진에서 내장 미리보기가 있을 경우 이를 이미지 처리에 우선 사용합니다. 색상 정확도가 향상될 수 있지만, 미리보기 품질은 카메라에 따라 달라지며 압축 아티팩트가 생길 수 있습니다.",
+    "image_prefer_wide_gamut": "광색역 우선 사용",
+    "image_prefer_wide_gamut_setting_description": "섬네일에 Display P3 색역을 사용합니다. 광색역 이미지를 생생하게 표현할 수 있지만, 구형 브라우저나 장치에서는 다르게 보일 수 있습니다. sRGB 이미지는 색상 왜곡 방지를 위해 그대로 유지됩니다.",
+    "image_preview_description": "단일 항목 보기 또는 기계 학습 시 사용되는 메타데이터 제거된 중간 크기 이미지",
```

```suggestion
    "image_preview_description": "단일 항목 보기 또는 기계 학습 시 사용되는 메타데이터가 제거된 중간 크기 이미지",
```
To make the sentence more natural, I added a conjunction.